.human/VIDEO_SCHEMA_STRATEGY.md

```
@@ -0,0 +1,29 @@
+# Estratégia de Schema para Vídeos (VideoObject)
+
+Para otimização avançada de SEO (AEO - Answer Engine Optimization) e para dominar o Painel de Conhecimento do Google (Knowledge Panel), os dados estruturados de vídeo (`VideoObject`) devem ser utilizados de forma estratégica.
+
+## O Risco do Schema Falso ou Invisível
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)